ga4gh/backend.py

```diff
@@ -374,10 +444,13 @@ def __init__(self, dataDir):
         for readGroupSetId in os.listdir(readGroupSetDir):
             relativePath = os.path.join(readGroupSetDir, readGroupSetId)
             if os.path.isdir(relativePath):
-                readGroupSet = reads.ReadGroupSet(
+                readGroupSet = reads.readGroupSetFactory(
```

Comment:
Why the factory? We might get rid of the factory for variantSets soon, so I wouldn't be too literal about following the conventions there.